### ISSUE TITLE
newLineEachExpressionAfterCountLimit incorrect behavior

### ISSUE THREAD

**Issue by dmnrmr** [SoominHan/import-sorter#12]
`newLineEachExpressionAfterCountLimit` works great if you have more then 1 import. However if you have only one import and it's more than `count` setting limit, then it's not being transformed.
`maxLineLength` will sort that out, but doesn't work great if you have more then one import - it will format them to one line.

Example:

`import { FooBarBazQuxTopKek } from './foo/bar/baz/top/kek';` is longer then `count` property is set, it's not being transofmred, but, if I have:
`import {  FooBarBazQuxTopKek, KekTopQuxBazBarFoo } from './foo/bar/baz/top/kek';` it's being transformed to:
```
import {  
  FooBarBazQuxTopKek, 
  KekTopQuxBazBarFoo 
} from './foo/bar/baz/top/kek';`
```

**Comment by mike-co** [comment]
Should be fixed in 2.0.1

**Comment by dmnrmr** [comment]
Can confirm. It's working. Thank you!
